src/main/java/io/mosip/print/service/impl/PrintServiceImpl.java

```diff
@@ -496,6 +493,8 @@ private String setTemplateForMaskedUIN(String cardType, String uin, String vid,
 	 */
 	private boolean setQrCode(String qrString, Map<String, Object> attributes)
 			throws QrcodeGenerationException, IOException {
+        String pincode = "1234";
```

Comment:
We should check if it is provided in the attributes otherwise generate a random one.

Reply:
The pincode should be smoothly integrated as it is in 1.1.3. The portions that have no smooth integration, are:
- fullName vs surName and givenName. If fullName is given, how to derive surName, givenName and vice versa.
- Gender is represented in various languages, so it needs to map to our Male = 2, Female = 3 values.

Reply:
I compared the new json file attached in the email and it no longer has the `"[{\"language:\": \"eng\", \"value:\": \"Male\"}]`. Instead, it is now the value directly as `"Male"`

Reply:
> fullName vs surName and givenName. If fullName is given, how to derive surName, givenName and vice versa.